mnd/mnd/forms/navigation/quick_links.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from rdrf.forms.navigation.quick_links import (
+    PromsMenuConfig, RegularMenuConfig, QuickLinks,
+    make_link, make_entries
+)
+
+
+MIMSProductCache = make_link("admin:mnd_mimsproductcache_changelist", _("MIMS product cache"))
+MIMSCMICache = make_link("admin:mnd_mimscmicache_changelist", _("MIMS CMI cache"))
+
+
+class MNDRegularMenuConfig(RegularMenuConfig):
+
+    def __init__(self, registries):
+        super().__init__(registries)
+        self.all.update(make_entries(MIMSProductCache, MIMSCMICache))
+
+
+class MNDQuickLinks(QuickLinks):
+
+    def __init__(self, registries):
```

Comment:
This is only a small amount of code duplication, but the fact that it's there makes me think QuickLinks + MenuConfig should be reorganised to better accommodate this usecase

Reply:
Updated.